### PR TITLE
add tests for helm utils

### LINKED ISSUE #986: Add e2e test for helm utils
Add e2e test cases for helm utils
## Expected Behaviour

internal dependencies version upgrades in future should not break anything working
## Current Behaviour

There is no test cases to identify if something broke due to version upgrade 
## If requesting a CLI for "arkade get"

How many downloads does this tool have? Check at: 

https://somsubhra.github.io/github-release-stats/

If approved, are you willing to submit a pull request for this?

NA

### PR DESCRIPTION
**Pull request by nitishkumar71**
## Description

Add tests for helm utils
## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
Closes #986

## How Has This Been Tested?

If updating or adding a new CLI to `arkade get`, run:

```
go build && ./hack/test-tool.sh TOOL_NAME
```
NA. As it's logic tests included , so nothing additional tests are needed to be done.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [ ] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: pkg/helm/io_test.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+				"fullnameOverride": "",
+			},
+		},
+		{
+			name:    "empty yaml file",
+			content: ``,
+			want:    ValuesMap{},
+		},
+		{
+			name: "invalid yaml file",
```
Comment (alexellis) [inline pkg/helm/io_test.go:43]: What's invalid about this YAML file, perhaps the test case name could say?

File: pkg/helm/io_test.go [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+func Test_Load(t *testing.T) {
+	tests := []struct {
+		name    string
+		content string
+		want    ValuesMap
+		err     bool
+	}{
+		{
+			name: "valid yaml file",
+			content: `
```
Comment (alexellis) [inline pkg/helm/io_test.go:18]: There should also be a test for when `image:` is at the top level of the file for things like inlets-operator.

File: pkg/helm/io_test.go (hunk 1 above)
Comment (nitishkumar71) [reply pkg/helm/io_test.go:43]:
```yaml
replicaCount: 1
image:
repository: nginx
```

not properly indented.

File: pkg/helm/io_test.go (hunk 2 above)
Comment (nitishkumar71) [reply pkg/helm/io_test.go:18]: updated content with a top level `image` key.

File: pkg/helm/io_test.go (hunk 1 above)
Comment (alexellis) [reply pkg/helm/io_test.go:45]: That looks like properly indented YAML to me?

But it sounds like you've come up with a great name for the test case.

Comment (nitishkumar71) [reply pkg/helm/io_test.go:45]: shouldn't valid yaml will look like below? There is no proper indentation. 
```
replicaCount: 1
  image:
repository: nginx
```

**Pull request closed without merging**